package.json

```diff
@@ -17,6 +17,7 @@
   },
   "dependencies": {
     "@types/dompurify": "3.0.2",
+    "@types/validator": "13.11.9",
```

Comment:
typing dependencies should be added to the `devDependencies` object instead of here.